server.py

```diff
@@ -369,6 +371,13 @@ def forbidden(e):
         else:
             return render_template("error/403.html"), 403
 
+    @app.errorhandler(429)
+    def forbidden(e):
+        if request.path.startswith("/api/"):
```

Comment:
can we add a `log.error` here please? We got paged when an error happens so this allows us to see either an attack is going on or is this rather a false positive.